src/software/primitive/primitive_msg_factory.h

```diff
@@ -0,0 +1,122 @@
+#pragma once
+
+#include "shared/proto/primitive.pb.h"
+#include "software/geom/angle.h"
+#include "software/geom/angular_velocity.h"
+#include "software/geom/point.h"
+#include "software/primitive/primitive.h"
+
+/**
+ * Enum for indicating the stop type for the Stop Primitive
+ */
+enum StopTypeMsg
```

Comment:
Msg is for protobuf messages
```suggestion
enum StopType
```

Reply:
Moved to `proto/primitive` and fixed.